staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json.go

```diff
@@ -32,6 +33,58 @@ import (
 	utilyaml "k8s.io/apimachinery/pkg/util/yaml"
 )
 
+var (
+	// Private copy of jsoniter.ConfigCompatibleWithStandardLibrary to try to shield against possible mutations
+	// from outside. Still does not protect from package level jsoniter.Register*() functions - someone calling them
+	// in some other library will mess with every usage of the jsoniter library in the whole program.
+	// See https://github.com/json-iterator/go/issues/265
+	configCompatibleWithStandardLibrary = jsoniter.Config{
```

Comment:
Do we use jsoniter elsewhere?  I.e. do we need our ow pkg/util/json that exposes this?

Reply:
My plan was to use `k8s.io/apimachinery/pkg/util/json` (which uses jsoniter) everywhere in k/k to get performance benefits across the board. See WIP https://github.com/kubernetes/kubernetes/pull/63284.